run.py

```diff
@@ -9,7 +9,7 @@
 import webpages
 import backend
 import logging
-from settings import repo_path, script_dir
+from settings_example import repo_path, script_dir, cluster_path
```

Comment:
`settings_example` is enkel als placeholder bedoelt. Het idee is dat je het gebruikt om een lokale `settings` te maken waar je conf. parameters in bewaart (mappen, wachtwoorden, etc.) die 1) anders zijn dan die op de server en 2) niet in een publieke repo terecht mogen komen (bijv. wachtwoorden).